radixdlt-core/radixdlt/src/main/java/com/radixdlt/keys/PersistedBFTKeyManager.java

```diff
@@ -68,4 +68,8 @@ public ECDSASignature sign(byte[] hash) {
 	public BFTNode self() {
 		return BFTNode.create(ecKeyPair.getPublicKey());
 	}
+
+	public ECKeyPair getKeyPair() {
```

Comment:
I don't think we should be exposing the private key in this way, better would be if exposed a Signer instead.